src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -240,6 +241,23 @@ export class WdsDataTableProvider implements DataTableProvider {
     );
   };
 
+  protected transformQuery = (query: string): string => {
+    const specialCharactersRegex = /([+\-&|!(){}[\]^"~*?:\\])/g;
+    return query.replace(specialCharactersRegex, '\\$1').replace('=', ':');
+  };
```

Comment:
a filter term containing spaces also causes query-parsing problems. That is solvable by double-quoting the entire search term … but then I fear that some of the escaping will be interpreted literally instead of as an escape sequence - we'll have to give it a try.

And, if there are backend changes we need to support the front end, let's do that!

Reply:
Ok I think including whitespace in the characters to escape works, with the exception of return lines (`\n`) not working if copied from the UI - I can see that the value returned from the database "test\nFoo\nBar", in the UI it looks like a space instead of a return, unless you inspect the element and can see the words on different lines - but if you then copy that value and search for it, nothing is returned - although there's at least no error.  Not sure if this is a problem with the copy or what.